CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to USWDS web components
+
+Thank you for your interest in contributing to USWDS web components.
+
+> [!Important]
+>
+> **This repo is currently in beta and is not yet ready for community code contributions.**
+>
+> While the USWDS core team works through big decisions about architecture and style, we ask that you do not submit pull requests. Instead, we encourage you to contribute by your sharing your ideas, concerns, questions, and experiences with the USWDS core team and community.
+>
+> Learn more about how you can provide feedback in the [submitting an issue](#submitting-an-issue) and [starting a discussion](#starting-a-discussion) sections below.
```

Comment:
**note:** Wondering if we should specifically call out here that we are interested in users doing beta testing.

Reply:
I like the idea, but will this get lost here? Are we messaging about seeking beta testers in other places?